browser/js/common/directives/kit-fields/kit-code-field.html

```diff
@@ -0,0 +1,9 @@
+<section id="kit-code-field" ng-controller="AceCtrl">
```

Comment:
Is there only ever going to be one of these directives on a given view? If not, this `id` could get you into trouble. If you're using it for CSS selection, maybe consider just selecting the directive by tag name, e.g. `#kit-code-field {...}` => `kit-code-field {...}`.